h/cli/commands/shell.py

```diff
@@ -75,5 +75,7 @@ def shell(config, type_):
 
     try:
         runner(**locals_)
-    except ImportError:
-        raise click.ClickException("The {!r} shell is not available.".format(type_))
+    except ImportError as e:
+        raise click.ClickException(
+            "The {!r} shell is not available.".format(type_)
+        ) from e
```

Comment:
I think having a variable this short is another lint violation. I think `err` does the trick. We might not want to have to go through this again later!